### PR TITLE
Handling new login window

### PR DESCRIPTION
**Pull request by ludwikkazmierczak**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/logintests/LoginTests.java [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import com.wikia.webdriver.common.core.Assertion;
 import com.wikia.webdriver.common.core.Helios;
 import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.RelatedIssue;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/logintests/LoginTests.java]: MINOR Remove this unused import 'com.wikia.webdriver.common.core.annotations.RelatedIssue'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 4 issues:
* MINOR 4 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Remove this unused import 'com.wikia.webdriver.common.core.WikiaWebDriver'.](https://github.com/Wikia/selenium-tests/blob/862e0ba6132f32528c0af02bd2e4aa0591f5c144/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/AuthModal.java#L3) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.common.core.elemnt.Wait'.](https://github.com/Wikia/selenium-tests/blob/862e0ba6132f32528c0af02bd2e4aa0591f5c144/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/AuthModal.java#L4) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'org.openqa.selenium.WebDriver'.](https://github.com/Wikia/selenium-tests/blob/862e0ba6132f32528c0af02bd2e4aa0591f5c144/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/AuthModal.java#L8) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Remove this unused import 'org.openqa.selenium.support.PageFactory'.](https://github.com/Wikia/selenium-tests/blob/b95546cfb80b7d0425a8a698dc01b70c77cb4245/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/AuthModal.java#L11) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by justnpT** [comment]
Verification check
http://jenkins:8080/view/Social/job/Test-Case-Automation-Login-Signup/16/

**Comment by ludwikkazmierczak** [comment]
LGTM

File: src/test/java/com/wikia/webdriver/testcases/logintests/LoginTests.java (hunk 1 above)
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/logintests/LoginTests.java]: MINOR Remove this unused import 'com.wikia.webdriver.common.core.annotations.RelatedIssue'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 5 issues:
* MINOR 5 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Remove this unused import 'com.wikia.webdriver.common.core.WikiaWebDriver'.](https://github.com/Wikia/selenium-tests/blob/b95546cfb80b7d0425a8a698dc01b70c77cb4245/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/AuthModal.java#L3) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.common.core.elemnt.Wait'.](https://github.com/Wikia/selenium-tests/blob/b95546cfb80b7d0425a8a698dc01b70c77cb4245/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/AuthModal.java#L4) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'org.openqa.selenium.support.PageFactory'.](https://github.com/Wikia/selenium-tests/blob/b95546cfb80b7d0425a8a698dc01b70c77cb4245/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/AuthModal.java#L11) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'org.openqa.selenium.WebDriver'.](https://github.com/Wikia/selenium-tests/blob/b95546cfb80b7d0425a8a698dc01b70c77cb4245/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/AuthModal.java#L8) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Pull request merged**